task_manager/users/admin.py

```diff
@@ -0,0 +1,12 @@
+import django_stubs_ext
+from django.contrib import admin
+from task_manager.users.models import ServiceUser
+
+django_stubs_ext.monkeypatch()
```

Comment:
it's better to do this monkeypatching only once in file like `settings.py`, like [here](https://github.com/wemake-services/wemake-django-template/blob/5bf1569e2710e11befc6991893f94419136d74bd/%7B%7Bcookiecutter.project_name%7D%7D/server/settings/__init__.py#L14-L19)